src/ChunkMap.h

```diff
@@ -246,6 +248,10 @@ class cChunkMap
 	Returns true if all block entities processed, false if the callback aborted by returning true. */
 	bool ForEachBlockEntityInChunk(int a_ChunkX, int a_ChunkZ, cBlockEntityCallback & a_Callback);  // Lua-accessible
 
+	/** Calls the callback for brewingstand in the specified chunk.
+	Returns true if all brewingstands processed, false if the callback aborted by returning true. */
+	bool ForEachBrewingstandInChunk(int a_ChunkX, int a_ChunkZ, cBrewingstandCallback & a_Callback);  // Lua-accessible
```

Comment:
This will only be an API function, but not an actual call in `cWorld` / `cChunkMap`, the API functions will use a common template-based framework over `ForEachEntityInChunk()`